raiden/tests/integration/transfer/test_directransfer.py

```diff
@@ -53,6 +53,7 @@ def test_direct_transfer_to_offline_node(raiden_network, token_addresses, deposi
     # Wait until the initialization of the node is complete and then stop it
     gevent.wait([app1.raiden.start_event])
     app1.raiden.stop()
+    gevent.sleep(2)
```

Comment:
The `_logout_timeout` in the Matrix transport is `0s by default. So this 2s sleep may not be long enough. Could we wait for an event instead of using a sleep?

Reply:
The logout is mostly synchronous, so it should already have happened here. This is just to ensure a little time before sending the message, to be sure to trigger the event when the node is offline. Not a hard requirement though, just a safer wait that was added at debugging times and I thought would be nice to be kept.